shopinvader_search_engine/models/se_backend.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SeBackend(models.Model):
+
+    _inherit = "se.backend"
+
+    primary_webshop_id = fields.Many2one("shopinvader.backend")
```

Comment:
I don't think we need these fields

Reply:
my idea was to have all fields and buttons available at least on the one form 
I can drop this or do computation related to sequence the main will be that one who has the engine set first